### DISCUSSION TITLE
release version scheme

### DISCUSSION THREAD

**Discussion by chenrui333** [Difegue/LANraragi#730] [General]
👋 it looks like the most recent release is `0.8.81`, not `0.8.9`, is there any documentations about the release scheme? Thanks!

relates to https://github.com/Homebrew/homebrew-core/pull/118898

**Comment by Difegue** [comment]
👋 `0.8.81` is a hotfix release for `0.8.8` as I didn't want to move to `0.8.9` yet.  
This is kinda similar to how some Perl packages version themselves, I'll probably have to make the next release `0.8.90` to compensate though. Sorry! 🙏

**Reply by chenrui333** [reply]
no worries, thanks for confirming it!! I wonder if it is good to document it somewhere. :)
